### ISSUE TITLE
Vanilla ElderJS Template generates redundant includes

### ISSUE THREAD

**Issue by muttoni** [Elderjs/elderjs#188]
Hi,

I was trying out the ElderJS template and just ran the default template setup (npx, npm install, npm start)

I then open the page and view source, I see lots (14) of duplicated includes:

```html

<!-- Line breaks added for clarity -->

<html lang="en"><head><meta charset="UTF-8" /><meta name="viewport" content="width=device-width, initial-scale=1" /><title>Elder.js Template: Home</title><meta name="description" content="Elder.js Starter Template: It's A Tutorial Too!" data-svelte="svelte-fpz8dk"><link href="/" rel="canonical" data-svelte="svelte-fpz8dk">
<link rel="stylesheet" href="https://unpkg.com/balloon-css/balloon.min.css" data-svelte="svelte-hprhv2">
<link rel="stylesheet" href="https://unpkg.com/balloon-css/balloon.min.css" data-svelte="svelte-hprhv2">
<link rel="stylesheet" href="https://unpkg.com/balloon-css/balloon.min.css" data-svelte="svelte-hprhv2">
<link rel="stylesheet" href="https://unpkg.com/balloon-css/balloon.min.css" data-svelte="svelte-hprhv2">
<link rel="stylesheet" href="https://unpkg.com/balloon-css/balloon.min.css" data-svelte="svelte-hprhv2">
<link rel="stylesheet" href="https://unpkg.com/balloon-css/balloon.min.css" data-svelte="svelte-hprhv2">
<link rel="stylesheet" href="https://unpkg.com/balloon-css/balloon.min.css" data-svelte="svelte-hprhv2">
<link rel="stylesheet" href="https://unpkg.com/balloon-css/balloon.min.css" data-svelte="svelte-hprhv2">
<link rel="stylesheet" href="https://unpkg.com/balloon-css/balloon.min.css" data-svelte="svelte-hprhv2">
<link rel="stylesheet" href="https://unpkg.com/balloon-css/balloon.min.css" data-svelte="svelte-hprhv2">
<link rel="stylesheet" href="https://unpkg.com/balloon-css/balloon.min.css" data-svelte="svelte-hprhv2">
<link rel="stylesheet" href="https://unpkg.com/balloon-css/balloon.min.css" data-svelte="svelte-hprhv2">
<link rel="stylesheet" href="https://unpkg.com/balloon-css/balloon.min.css" data-svelte="svelte-hprhv2">
<link rel="stylesheet" href="https://unpkg.com/balloon-css/balloon.min.css" data-svelte="svelte-hprhv2">
<link rel="stylesheet" href="/_elderjs/assets/svelte-e55d64fd.css" media="all" /><style></style></head><body class="home"><div class="container svelte-1tkpvyy">


<!-- rest of code -->
```

Are you able to replicate as well?

While I'm aware that the browser will probably ignore the duplicate includes, and that the culprit css file is an optional hover effect originating from HookDetail.svelte, if you are able to replicate the behavior it may make sense to check for & remove duplicates in the build process.

**Comment by nickreese** [comment]
@muttoni Thanks for bringing this up. Looks like it is happening on the production template too. Will investigate, this must be a recent bug.

**Comment by nickreese** [comment]
@muttoni Happy to report that this is user error and not an Elder.js bug. :)

When refactoring the template, I moved the loading of the `https://unpkg.com/balloon-css/balloon.min.css` into the `<svelte:head>` within `HookDetail.svelte` which is called N number of times on the homepage... resulting in it getting added N times. 

I've revised the template. It is now in `Layout.svelte` which is less than ideal as it is loaded on all pages... but it will be less likely to cause issues as a user only needs to delete it one place.
